### PR TITLE
(SIMP-2922) Updated Kerberos HOWTO

### PR DESCRIPTION
**Pull request by trevor-vaughan**
SIMP-2922 #close

### CODE REVIEW

File: docs/user_guide/HOWTO/Kerberos.rst [hunk 1]
```diff
@@ -24,21 +35,19 @@ The following sections give a brief guide on how to get started, for more
 information, please see the `official Red Hat documentation`_.
 
 .. NOTE::
-  You can skip this section if you're using the default settings. These will
-  complete the following for you with randomly generated passwords for all
-  keytabs and the master password.
 
-Usage
------
+   You can skip this if you're using the default settings. These will complete
```
Comment (nick-markowski) [inline docs/user_guide/HOWTO/Kerberos.rst]: This is confusing.  It's hard to tell what you can 'skip' and what you can't.

File: docs/user_guide/HOWTO/Kerberos.rst [hunk 2]
```diff
@@ -96,19 +105,19 @@ Propagate the Keytabs
 ^^^^^^^^^^^^^^^^^^^^^
 
 Move all of the resulting keytab files SECURELY to
-`<environment_dir>/keydist/<fqdn>/keytabs` on the Puppet server as appropriate
-for each file.
+``/var/simp/environments/<client_environment>/site_files/krb5_files/keytabs/<fqdn>``
```
Comment (nick-markowski) [inline docs/user_guide/HOWTO/Kerberos.rst]: I think this is /var/simp/environments/<client_environment>/site_files/krb5_files/**files**/keytabs/fqdn

File: docs/user_guide/HOWTO/Kerberos.rst [hunk 3]
```diff
@@ -60,7 +69,7 @@ Run the following command, as root, to create your principal:
 
   # /usr/sbin/kadmin.local -r YOUR.DOMAIN -q "addprinc <username>/admin"
```
Comment (nick-markowski) [inline docs/user_guide/HOWTO/Kerberos.rst]: kadmin.local is provided by krb5-server, which is not installed by default. Might want to notify them.

Comment (trevor-vaughan) [inline docs/user_guide/HOWTO/Kerberos.rst]: @nick-markowski After you've run `puppet`, `kadmin.local` should be installed.

**Pull request merged**